linux/user-and-file-management/user-permissions/file-ownership.md

```diff
@@ -42,33 +28,26 @@ On Linux, every file and directory is given both a user owner and a group owner.
 Here is some example output:
 
 ```bash
-$ ls -ld /var/log
+ls -ld /var/log
 drwxrwxr-x 10 root syslog
     4096 Jun 13 06:25 /var/log
-$ ls -l /var/log/syslog
+ls -l /var/log/syslog
 -rw-r
-
 ```
 
 ---
 ## Practice
 
-Consider the following snippet:
+How would you find out the owners of the `timetable.txt` file?
```

Comment:
why change this? did users complain?

(from what i see the insight is not talking about how `-r-xr...` translates into group permissions, but the previous insight might do it, such that actually have the question here isn't that bad)

Reply:
yes, users complained that the question has nothing to do with the insight (insight talks about ownership while the question references permissions)

there is no previous insight in which we explain permissions (in this workout or another one), so it was impossible for users to answer

Reply:
![image](https://user-images.githubusercontent.com/19802557/84653405-4412c700-af05-11ea-8e18-def2559ecc60.png)

![image](https://user-images.githubusercontent.com/19802557/84653436-4ecd5c00-af05-11ea-862b-430d16587e20.png)

![image](https://user-images.githubusercontent.com/19802557/84653449-555bd380-af05-11ea-89e3-68f22228e361.png)